### PR TITLE
script to add admin user to database

### PR DESCRIPTION
**Pull request by beblicarl**
#### Pre-Submission Checklist

-   [x] Your pull request targets the `main` branch.
-   [x] Branch name is descriptive and follows the guidelines [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices).
-   [x] All tests pass . Use `git commit --amend` to amend any fixes.

#### Checklist:

-   [x] All tests are passing.
-   [x] Tested changes locally.
-   [x] Improved documentation or readme where necessary.
-   [x] Your commit messages format follows our commit message guideline [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices#commits).

#### What does this PR do?
script to add admin user to database

#### How should this be manually tested?

1. Insert the data by running this command `node src/scripts/adminuser`

#### What are the relevant issues?
script to add admin user to database

### CODE REVIEW

File: src/scripts/adminuser.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const db = require("../db")
```
[the remark this answers is not in the document]
Comment (beblicarl) [reply src/scripts/adminuser.js:1]: alright, thanks

File: src/scripts/adminuser.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+const insertQuery = async() => {
```
[the remark this answers is not in the document]
Comment (beblicarl) [reply src/scripts/adminuser.js:3]: okay, i will take note

File: src/scripts/adminuser.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	await db.query(
```
[the remark this answers is not in the document]
Comment (beblicarl) [reply src/scripts/adminuser.js:4]: yes please

File: package.json [hunk 4]
```diff
@@ -29,7 +29,7 @@
     "husky": "^8.0.0",
     "minimist": "1.2.6",
     "mocha": "^10.0.0",
-    "ndb": "1.1.5",
```
Comment (jocrah) [inline package.json:32]: @beblicarl please why this change?

Comment (beblicarl) [reply package.json:32]: I was told to update my npm , mongo and everything else including force audit for the error I was facing to go but it still didn't. I'm sure it's one of the things I installed

Comment (jocrah) [reply package.json:32]: not sure how an old version helps but cool

**Review by jocrah — APPROVED** [review]
@beblicarl i'm merging this for now. I'm thinking the script should have a beginning phase where it connects to the database like the `seeddb.js` does but curious to know how it works out in production

so kindly do not close the related issue yet. will give you feedback on how it goes

**Comment by beblicarl** [comment]
> @beblicarl i'm merging this for now. I'm thinking the script should have a beginning phase where it connects to the database like the `seeddb.js` does but curious to know how it works out in production
> 
> so kindly do not close the related issue yet. will give you feedback on how it goes

No problem at all, i patiently await your feedback.
Any strange behavior in the code is your fault though haha

**Comment by jocrah** [comment]
> No problem at all, i patiently await your feedback. Any strange behavior in the code is your fault though haha

haha got it. i will congratulate myself when it goes well too
